builtin/providers/clc/resource_clc_server.go

```diff
@@ -44,15 +44,14 @@ func resourceCLCServer() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			// optional
 			"password": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
@stack72 I was under the impression that the `d.Get` on line 125 will return `""` if not provided... 

That's based on: https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_data.go#L48-L60